src/main/webapp/app/tasks/task.ts

```diff
@@ -4,4 +4,5 @@
 export interface Task {
   id: string;
   name: string;
+  isComplete ?: boolean;
```

Comment:
You can assume that you are not working with legacy data. So make this field non-optional.